README.md

```diff
@@ -121,8 +121,8 @@ During development, Dockerfile frontend (dockerfile.v0) is also part of the Buil
 ##### Building a Dockerfile with `buildctl`
 
 ```
-buildctl build --frontend=dockerfile.v0 --local context=. --local dockerfile=.
-buildctl build --frontend=dockerfile.v0 --local context=. --local dockerfile=. --frontend-opt target=foo --frontend-opt build-arg:foo=bar
+buildctl build --frontend=type=dockerfile.v0 --local context=. --local dockerfile=.
+buildctl build --frontend=type=dockerfile.v0,target=foo,build-arg:foo=bar --local context=. --local dockerfile=.
```

Comment:
`build-arg:foo=bar` looks odd here; is that because `build-arg` is part of `target`?

Should this there something like

```
opts=target=foo,build-arg=foo=bar` 
```

or

```
opt=target=foo,opt=build-arg=foo=bar
```

🤔

Reply:
Actually ^^ I think we may _have to_ use an `opt` (or whatever) field for that, otherwise a frontend that defines a `type=` option will conflict with the built-in `type=`

Reply:
The `:` comes from internal data type requirements. It's a map where key is `build-arg:foo` and not `build-arg`. Not using `=` makes parsing not use the wrong key (both here and in old single flag format).

Reply:
@thaJeztah for the record: the csv style allows for grouping with `"`: `build-arg="type=foo"` so that `type=` would not get misinterpreted.

Reply:
The issue with `type` is not with build args. A frontend gets access to all options. It would be weird to define that they can't use a key "type" because that is used in cli parsing.